deployment/monocle-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+# created manually using the command: 
+# oc create configmap --from-file=config.yaml monocle-config --dry-run=client -oyaml
```

Comment:
Is this comment valueable for understanding the code? If not, then it can be added to the commit comment or PR description.

Reply:
It is important when re-creating that resource, as the generation command currently is a bit manual